config/routes/api/v2.rb

```diff
@@ -206,7 +206,9 @@
 
       resources :http_proxies, :except => [:new, :edit]
 
-      resources :trends, :only => [:create, :index, :show, :destroy]
+      # TODO: remove in 2.4
+      statistics_plugin = Foreman::Plugin.find(:foreman_statistics)
+      resources :trends, :only => [:create, :index, :show, :destroy], :controller => statistics_plugin && '/foreman_statistics/api/v2/trends'
```

Comment:
won't this prevent getting to the core controller if the plugin isn't there?

Reply:
no, it will be `controller: nil` and it will be like it's not there. Promise, I've tested it and the tests are testing it as well :thinking: